model/modeldecoder/context.go

```diff
@@ -124,16 +124,22 @@ func decodeURL(raw common.MapStr, err error) (*model.URL, error) {
 	return &url, err
 }
 
-func getHTTPClientIP(http *model.Http) net.IP {
+func getHTTPClientIP(http *model.Http, labels *model.Labels) net.IP {
```

Comment:
Instead of passing ctx.Lables to determine RUM vs Trace flow, you can pass the context itself and use it ctx.IsRum

Reply:
This is not possible.

In the middleware go file, ctx is of type request.Context, which has the field isRum.

In this file, ctx is of type model.Context, which does not have the field. Hence attempting to do it will give a syntax error.